themes/buildpacks/layouts/shortcodes/comparison-table.html

```diff
@@ -154,16 +193,26 @@
     <tr>
       <th class="p-3">Governance</th>
       <td class="border-left border-right">CNCF</td>
+      <td>Docker</td>
       <td>Red Hat</td>
       <td>Google</td>
       <td>Google</td>
     </tr>
+    <tr class="border-top bg-light">
+      <th class="p-3">Target image types</th>
+      <td class="border-left border-right">Applications</td>
+      <td class="">Applications, Base images, Operating Systems</td>
```

Comment:
```suggestion
      <ul class="list-circle">
          <li>Applications</li>
          <li>Base Images</li>
          <li>Operating Systems</li>
        </ul>
```

Given it's a list I think it would look better formatted as such.